app/src/main/java/com/geeksville/mesh/service/MeshService.kt

```diff
@@ -1931,7 +1931,25 @@ class MeshService : Service(), Logging {
                 removeByNodenum = nodeNum
             })
         }
-
+        override fun requestUserInfo(packetId: Int, destNum: Int, sourceNodeInfo: NodeInfo) = toRemoteExceptions {
```

Comment:
we can simplify this a bit. we only need `destNum` as parameter since `packetId` is unused and the service has the nodeDB with our user info (already as MeshProtos.User).